hackathon/templates/hackathon/hackathon_view_public.html

```diff
@@ -23,7 +23,11 @@
 
                 <div class="row">
                     <div class="col p-0">
-                        <a role="button" class="btn btn-ci mt-3 w-100" href="{% url 'account_login' %}">Login To Register</a>
+                        {% if not hackathon.is_register %}
+                            <a role="button" class="btn btn-ci mt-3 w-100" href="https://forms.gle/Hp7N8RF6LzyPHobi7" target="_blank">Register your interest</a>
```

Comment:
Could make the form configurable through an environment variable? The link might change over time and we probably don't want to create a new PR every time.